solutions/pytorch_tests/Dockerfile-build

```diff
@@ -0,0 +1,61 @@
+# Adapted from https://github.com/pytorch/pytorch/blob/master/Dockerfile
+ARG BASE_IMAGE=ubuntu:18.04
+ARG PYTHON_VERSION=3.8
```

Comment:
Nit: `PYTHON_VERSION` and `PYTORCH_VERSION` is extraneous as it isn't used, and lost after the next `FROM` directive. You should set these args and defaults after the `FROM` where you do refer to the variable (e.g. after the `FROM` on line 22 for `PYTHON_VERSION` and after the line 32 for `PYTORCH_VERSION`)

Also the same case for `solutions/pytorch_tests/Dockerfile`

Reply:
Fixed.